pkg/controller/operators/suite_test.go

```diff
@@ -0,0 +1,147 @@
+package operators
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/operator-framework/api/crds"
+	operatorsv2alpha1 "github.com/operator-framework/api/pkg/operators/v2alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/storage/names"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/reference"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	"sigs.k8s.io/controller-runtime/pkg/envtest/printer"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	// +kubebuilder:scaffold:imports
+
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/controller/operators/decorators"
+)
+
+// These tests use Ginkgo (BDD-style Go testing framework). Refer to
+// http://onsi.github.io/ginkgo/ to learn more about Ginkgo.
+
+const (
+	timeout  = time.Second * 10
+	interval = time.Millisecond * 100
+)
+
+var (
+	cfg       *rest.Config
+	k8sClient client.Client
+	testEnv   *envtest.Environment
+	scheme    = runtime.NewScheme()
+
+	gracePeriod int64 = 0
+	propagation       = metav1.DeletePropagationForeground
+	deleteOpts        = &client.DeleteOptions{
+		GracePeriodSeconds: &gracePeriod,
+		PropagationPolicy:  &propagation,
+	}
+	genName = names.SimpleNameGenerator.GenerateName
+)
+
+func TestAPIs(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	RunSpecsWithDefaultAndCustomReporters(t,
+		"Controller Suite",
+		[]Reporter{printer.NewlineReporter{}})
+}
+
+var _ = BeforeSuite(func() {
```

Comment:
does this conflict with the test runner that Ben worked on in (see: `x.mk`)?

Reply:
I don't think so. This is scoped to the suite defined in this package, not to mention that the test sets options for the e2e tests and not unit tests.